docs/HowToUsePeripheral.md

```diff
@@ -186,7 +200,7 @@ void up_serialinit(void)
 ### Add UART device structure
 The ```CONSOLE_DEV``` and ```TTYSx_DEV``` are UART device driver structure type named *uart_dev_t*.  
```

Comment:
How about rephrasing this as "uart_dev_t" is a generic UART device driver structure that represents CONSOLE_DEV and TTYSx_DEV at runtime?